File: Java Script - Introduction to Programming/3.1. Simple Conditions/13. Area of Figures.js
function myFunc([str, a , b]) {
  switch (str) {
    case 'square': return ((a * a).toFixed(3))
    case 'rectangle': return ((a * b).toFixed(3))
    case 'circle': return ((a * a * Math.PI).toFixed(3))
    case 'triangle': return ((a * b / 2).toFixed(3))
    default: throw ('Error!')
  }
}

//console.log(myFunc(['circle', '6']))

File: Pyrhon - Introduction to Programming/3.1. Simple Conditions/13. Area of Figures.py
figure = input()
if figure == 'square' or figure == 'circle':
    a = float(input())
else:
    a = float(input())
    b = float(input())

result = 0

if figure == 'square':
  result = (a ** 2)
elif figure == 'rectangle':
  print('hi')
  result = a * b
elif figure == 'circle':
  result = a * a * 3.14159
elif figure == 'triangle':
  result = a * b / 2

print(f'{result:.3f}')

